sample_4.py

```diff
@@ -0,0 +1,15 @@
+class Solution:
+    def fib(self, N: int) -> int:
+        # iterative O(n) time O(1) space
```

Comment:
add if statement to check whether number is int or not-
isinstance(N, int)

Reply:
Modified to return None when input is not of Integer type.

Reply:
I think bits not required, by default bit would return none only